lwt/nbd_lwt_unix.ml

```diff
@@ -76,6 +76,7 @@ let generic_channel_of_fd fd role =
   let ch = cleartext_channel_of_fd fd role
   in return (Channel.generic_of_cleartext_channel ch)
 
+(* This function is used by the client. The channel has no TLS ability. *)
```

Comment:
1. Yes, the returned channel. I don't see anything else it could refer to.
2. Not essential, but I felt it was helpful.